src/prefect/cli/flow.py

```diff
@@ -39,3 +46,107 @@ async def ls(
         )
 
     app.console.print(table)
+
+
+@flow_app.command()
+async def serve(
+    entrypoint: str = typer.Argument(
+        None,
+        help=(
+            "The path to a file containing a flow and the name of the flow function in"
+            " the format `./path/to/file.py:flow_func_name`."
+        ),
+    ),
+    name: str = typer.Option(
+        ...,
+        "--name",
+        "-n",
+        help="The name to give the deployment created for the flow.",
+    ),
+    description: Optional[str] = typer.Option(
+        None,
+        "--description",
+        "-d",
+        help=(
+            "The description to give the created deployment. If not provided, the"
+            " description will be populated from the flow's description."
+        ),
+    ),
+    version: Optional[str] = typer.Option(
+        None, "-v", "--version", help="A version to give the created deployment."
+    ),
+    tags: Optional[List[str]] = typer.Option(
+        ...,
+        "-t",
+        "--tag",
+        default_factory=list,
+        help="One or more optional tags to apply to the created deployment.",
+    ),
+    cron: Optional[str] = typer.Option(
+        None,
+        "--cron",
+        help=(
+            "A cron string that will be used to set a schedule for the created"
+            " deployment."
+        ),
+    ),
+    interval: Optional[int] = typer.Option(
+        None,
+        "--interval",
+        help=(
+            "An integer specifying an interval (in seconds) between scheduled runs of"
+            " the flow."
+        ),
+    ),
+    interval_anchor: Optional[str] = typer.Option(
+        None, "--anchor-date", help="The start date for an interval schedule."
+    ),
+    rrule: Optional[str] = typer.Option(
+        None,
+        "--rrule",
+        help="An RRule that will be used to set a schedule for the created deployment.",
+    ),
+    timezone: Optional[str] = typer.Option(
+        None,
+        "--timezone",
+        help="Timezone to used scheduling flow runs e.g. 'America/New_York'",
+    ),
+    pause_on_shutdown: bool = typer.Option(
+        True,
```

Comment:
Is this flag set to false via `--pause-on-shutdown=false` ?

Reply:
You can set it to false with `--no-pause-on-shutdown` which is Typer's default way of handling boolean flags. We can change it if it's too verbose.

Reply:
Ah sweet, no need just making sure I understand